src/test/java/org/apache/sysds/test/functions/federated/multitenant/FederatedMultiTenantTest.java

```diff
@@ -104,11 +104,10 @@ public void testSumSameWorkersSP() {
 		runMultiTenantSameWorkerTest(OpType.SUM, 4, ExecMode.SPARK);
 	}
 
-//FIXME still runs into blocking
-//	@Test
-//	public void testSumSharedWorkersSP() {
-//		runMultiTenantSharedWorkerTest(OpType.SUM, 3, 9, ExecMode.SPARK);
-//	}
+	@Test
+	public void testSumSharedWorkersSP() {
+		runMultiTenantSharedWorkerTest(OpType.SUM, 3, 9, ExecMode.SPARK);
+	}
```

Comment:
@mboehm7 I wasn't able to reproduce the error you mentioned. Is the blocking state of this test caused by the not set _DMLScript.USE_LOCAL_SPARK_CONFIG_ flag or is there another problem? :thinking: